SX1276/SX1276_LoRaRadio.cpp

```diff
@@ -848,29 +844,24 @@ void SX1276_LoRaRadio::standby( void )
  * and finally a DIO0 interrupt let's the state machine know that a packet is
  * ready to be read from the FIFO
  */
-void SX1276_LoRaRadio::receive(uint32_t timeout)
+void SX1276_LoRaRadio::receive(uint32_t /**timeout**/)
```

Comment:
Does this make it a Doxygen comment? Is that deliberate?

Reply:
I am not sure if this makes a doxygen comment. But it is deliberate. I just wanted to emphasis that this parameter is not being used. We will deprecate the APIs in a separate commit.

Reply:
If you aren't consciously doing something for Doxygen, then probably best not to use the special `/**` Doxygen syntax. Then we won't have to worry about how it will interpret it.